deps/build.jl

```diff
@@ -12,7 +12,19 @@ end
 ROOTENV = get(ENV, "CONDA_JL_HOME", DefaultDeps.ROOTENV)
 MINICONDA_VERSION = get(ENV, "CONDA_JL_VERSION", DefaultDeps.MINICONDA_VERSION)
 
-deps = """# Generated from $(@__FILE__) on $(now())
+if isdir(ROOTENV) && MINICONDA_VERSION != DefaultDeps.MINICONDA_VERSION
+    error("""Miniconda version changed, since last build.
+However, an Root Enviroment already exists at $(ROOTENV).
+Setting Miniconda version is not supported for existing Root Enviroments.
+To leave Miniconda version as it is unset the CONDA_JL_VERSION enviroment variable, and rebuild.
+To change Miniconda version, you must delete the Root Enviroment, and rebuild.
+WARNING: deleting the root enviroment will delete all the packages in it.
+This will break any Julia packages that have used Conda to install their dependancies.
+These will require rebuilding.
```

Comment:
Not necessarily.  If a PyCall-based package used `pyimport_conda("foo")`, then the package will automatically re-install the next time that package is used.

Maybe, "Some Julia packages that use Conda to install their dependencies may need to be re-built."